ui/pages/import-token/import-token.component.js

```diff
@@ -501,7 +501,13 @@ class ImportToken extends Component {
           id="custom-symbol"
           label={
             <div className="import-token__custom-symbol__label-wrapper">
-              <span className="import-token__custom-symbol__label">
+              <span
+                className={
+                  symbolAutoFilled && !forceEditSymbol
+                    ? 'import-token__custom-symbol__label--disabled'
+                    : 'import-token__custom-symbol__label'
+                }
+              >
```

Comment:
I believe we want the __label styles always applied, conditionally adding a second class with the --disabled modified. Otherwise this span doesn't get the 'flex: 0 0 auto'; from the plain __label class name.

Reply:
Understood, I will update this.

Reply:
```
            <span
                className={
                  'import-token__custom-symbol__label ' +
                  (symbolAutoFilled && !forceEditSymbol)
                    ? 'import-token__custom-symbol__label--disabled'
                    : ''
                }
              >
```
does this look good?

Reply:
I have not done a lot of scss but I have defined it like this,

Reply:
this has some details https://dev.to/mikkel250/getting-started-with-sass-nesting-and-parent-selectors-4617

But the `&` is computed as the parent class's name, so if &__label is nested under `import-token__custom-symbol` then a new css rule is created:
```css
import-token__custom-symbol__label {
   flex: 0 0 auto;
}
```

the --disabled modifier will also get its own styles. In order to have both the element must have both class names. In the example link you'll notice they do this:

```scss
.parent {
  width: 100%;
  &.new-child {
    height: 100%;
  }
}
```
the `&.` part is essential here because when you add the additional '.' it computes to two selectors by class name which still both must be present on to target the class. 

```css
.parent {
  width: 100%;
}

.parent.new-child {
  height: 100%;
}
```

hope that helps

Reply:
Thanks for such a detailed explanation, I will try to fix these asap.

Reply:
I have made the changes let me know if I need to add anything else.